### PR TITLE
nix-command and gerbsup updates

### PR DESCRIPTION
**Pull request by AlexKnauth**

### CODE REVIEW

File: scripts/update-gerbil-nix-recipe.ss [hunk 1]
```diff
@@ -47,22 +50,42 @@
...
+           (let ((d1 (path-expand reponame checkouts-dir))
+                 (d2 (path-expand reponame (path-expand owner (default-checkouts-by-owner-dir)))))
+             (if (or (file-exists? d1) (not (file-exists? d2))) d1 d2)))))
   (defvalues (repo-url site owner repo-name git-tree) (parse-repo-argument repo))
+  (def source-dir (defaultize source-dir_ owner repo-name))
+  (def nixpkgs-dir (defaultize nixpkgs-dir_ "NixOS" "nixpkgs"))
   (def recipe-file (path-expand recipe-path nixpkgs-dir))
 
+  (unless (file-exists? source-dir)
+    (error/dir-not-found name source-dir (filter (cut string-contains <> name) cmdopts)))
```
Comment (fare) [inline scripts/update-gerbil-nix-recipe.ss]: I don't understand the string-contains logic—is that a debug tool to check that the options are well-named? That deserves a comment.

File: scripts/update-gerbil-nix-recipe.ss [hunk 2]
```diff
@@ -15,6 +15,9 @@
 (defonce (default-checkouts-dir)
   (path-maybe-normalize (path-expand "../.." (path-directory (this-source-file)))))
```
Comment (fare) [inline scripts/update-gerbil-nix-recipe.ss:17]: I propose you use the new `(this-source-directory)`

File: scripts/update-gerbil-nix-recipe.ss [hunk 3]
```diff
@@ -274,7 +307,8 @@
       checkouts-dir: checkouts-dir
       source-dir: smug-gerbil-dir
       nixpkgs-dir: nixpkgs-dir
-      stable: stable))
+      stable: stable
+      cmdopts: ["--smug-gerbil-dir" "--gerbil-off" "--stable"]))
```
Comment (fare) [inline scripts/update-gerbil-nix-recipe.ss]: While we're at it, shouldn't we uniformly add the -repo and -dir arguments everywhere?

**Review by fare — APPROVED** [review]
LGTM

File: scripts/update-gerbil-nix-recipe.ss (hunk 2 above)
Comment (AlexKnauth) [reply scripts/update-gerbil-nix-recipe.ss:17]: Oh, thanks, I'll do that.

File: scripts/update-gerbil-nix-recipe.ss (hunk 3 above)
Comment (AlexKnauth) [reply scripts/update-gerbil-nix-recipe.ss]: Sure I can do that.

File: scripts/update-gerbil-nix-recipe.ss (hunk 1 above)
Comment (AlexKnauth) [reply scripts/update-gerbil-nix-recipe.ss]: I was thinking the `cmdopts` might contain options that apply to multiple things: the source-dir, the checkout-dir, and the nixpkgs-dir. I ended up not doing that, so I can change it back and clarify.

**Comment by AlexKnauth** [comment]
Okay, I have changed those three things:
 * Used `(this-source-directory)`,
 * Removed filter of `cmdopts` and added a comment to clarify its meaning,
 * Added `-repo` command-line options for everything that didn't have them already to be consistent.

**Pull request merged**